server/router/router.go

```diff
@@ -0,0 +1,27 @@
+package router
+
+import (
+	"app/controllers"
+
+	"github.com/gin-gonic/gin"
+)
+
+func InitRouter() {
+
+	r := gin.Default()
+
+	router := r.Group("/v1")
+	{
+		// users
+		user_router := router.Group("/user")
```

Comment:
In Go, camel case is the preferred convention.
https://go.dev/doc/effective_go#mixed-caps

Reply:
@Julius-R updated variable to camelCase